configs/j784s4_evm_gp_all_defconfig

```diff
@@ -0,0 +1,13 @@
+# J784s4 EVM configuration for GP devices
```

Comment:
NO. Delete this - GP should never exist.

Reply:
Sorry, but requesting a bit of clarification. We do have GP configuration for [J721S2-EVM](https://github.com/nmenon/k3-upstream-boot-build/blob/main/configs/j721s2_evm_gp_all_defconfig#L1) and [J721E-EVM](https://github.com/nmenon/k3-upstream-boot-build/blob/main/configs/j721e_evm_gp_all_defconfig#L1) in the repo. Are we not trying to do the same for J784S4-EVM? I could be totally wrong, but we do test J784S4 GP EVM configurations in the test-farm internally, right?

Reply:
J721e came out before we started going all in on HS devices, so they get a pass (for now). All new devices should be HS only from the start.